geode-assembly/src/integrationTest/resources/dependency_classpath.txt

```diff
@@ -1,5 +1,9 @@
 HikariCP-3.2.0.jar
 antlr-2.7.7.jar
+asm-6.2.jar
```

Comment:
why add logging will require all these dependencies?

Reply:
Because this is switching to java-based configuration for the web.xml. I'm not convinced this is the best approach, but it does give more flexibility around config - especially being able to enable or disable it programmatically.